specs/ipyplotly_integration/Overview.ipynb

```diff
@@ -0,0 +1,909 @@
+    "    scatt2.marker.color = selected"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 45,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create reset button\n",
+    "button = Button(description=\"clear\")\n",
+    "button.on_click(reset_brush)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 46,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Hide colorbar for figure 1\n",
+    "scatt1.marker.showscale = False"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 47,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set dragmode to lasso for both plots\n",
+    "f1.layout.dragmode = 'lasso'\n",
+    "f2.layout.dragmode = 'lasso'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 50,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Display two figures and the reset button\n",
+    "f1.layout.width = 500\n",
+    "f2.layout.width = 500"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 49,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "74f7b16952584919b650fbc442c21773",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/html": [
+       "<p>Failed to display Jupyter Widget of type <code>VBox</code>.</p>\n",
+       "<p>\n",
+       "  If you're reading this message in the Jupyter Notebook or JupyterLab Notebook, it may mean\n",
+       "  that the widgets JavaScript is still loading. If this message persists, it\n",
+       "  likely means that the widgets JavaScript library is either not installed or\n",
+       "  not enabled. See the <a href=\"https://ipywidgets.readthedocs.io/en/stable/user_install.html\">Jupyter\n",
+       "  Widgets Documentation</a> for setup instructions.\n",
+       "</p>\n",
+       "<p>\n",
+       "  If you're reading this message in another frontend (for example, a static\n",
+       "  rendering on GitHub or <a href=\"https://nbviewer.jupyter.org/\">NBViewer</a>),\n",
+       "  it may mean that your frontend doesn't currently support widgets.\n",
+       "</p>\n"
+      ],
+      "text/plain": [
+       "VBox(children=(HBox(children=(FigureWidget(), FigureWidget())), Button(description='clear', style=ButtonStyle())))"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "VBox([HBox([f1, f2]), button])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 55,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Save figure 2 to a svg image in the exports directory\n",
+    "f2.save_image('exports/f2.svg')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 54,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Save figure 1 to a pdf in the exports directory (requires cairosvg be installed)\n",
+    "# f1.save_image('exports/f1.pdf')"
```

Comment:
After installing cariosvg I get an error running this cell:

```
OSError: dlopen() failed to load a library: cairo / cairo-2
```

Looks like there's a thread open here: https://github.com/Kozea/CairoSVG/issues/84

Reply:
Did you install through pip or conda? I've had better luck through conda as conda will also pull down the cairo binary without needing to build it.  It might be that you have cairosvg installed but not cairo, in this case you might be able to install just cairo using conda.

Reply:
I think it's important that the install works w/ pip (as well as conda)

Reply:
very possible.

and yes I was using pip

Reply:
@cldougl Agreed. The trouble is that cairsosvg requires the non-python cairo executable be installed and pip can't (or at least doesn't) do that.

If cairo(svg) proves to be too much of a hassle to be even an optional dependency then one option is to allow users to register custom file type output filters.  This way a user can register `cairosvg.svg2png` to be the function to be called when a plot is to be saved to a `png` file.